src/middleware.ts

```diff
@@ -3,3 +3,4 @@ export * from './middleware/devtools'
 export * from './middleware/subscribeWithSelector'
 export * from './middleware/combine'
 export * from './middleware/persist'
+export * from './middleware/immer'
```

Comment:
This is controversial. I have been hesitating to provide this, but maybe people want it... (hm, I think we already discussed about this somewhere?)

Reply:
I think some people already copy paste immer middleware from the tests, it'd be hard from them to migrate or even author the middleware themselves so I think this is a fair addition.

Reply:
yeah, it's convincing...